include/core/interface_tools.h

```diff
@@ -286,6 +304,414 @@ namespace InterfaceTools
                                       &interface_reconstruction_cells,
     std::set<types::global_dof_index> &intersected_dofs);
 
+  /**
+   * @brief Solver to compute the signed distance from a given level of a level-set field. It is based on the method proposed by Ausas (2010).
+   *
```

Comment:
Can you put the DOI of the reference here. The year / Author is often not enough to find the pape.r